.github/workflows/container_reg.yml

```diff
@@ -0,0 +1,46 @@
+name: Build and push image to GitHub Container Registry
+on:
+  push:
+    tags:
+      - 'v*.*.*' # i.e. has to have a release tag to run 
```

Comment:
Could be nice to add a workflow dispatch here so that you can run the workflow manually, to deploy test images etc. 
see [this link](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch)

Reply:
Thanks, I didn't know about this option!   
I have now added a commit 4a98b6cac55128e1ab678b0c8ca0db65226e584f to add this in. And put a parameter in to control the docker tags. Does that look good? 

It seems the PR must be merged before I can test the workflow_dispatch option unfortunately (I can't see it in the actions sidebar at least).

Reply:
Yes. I believe that the workflow with the dispatch must reside in the default branch.

Reply:
Thanks, I will test the build once the PR is merged then and update it if needed.